theano/configdefaults.py

```diff
@@ -361,6 +361,22 @@ def default_dnn_library_path():
              EnumStr("auto", "True", "False", "no_check"),
              in_c_key=False)
 
+AddConfigVar('magma.include_path',
+             "Location of the magma header",
+             StrParam(None),
```

Comment:
I think None should be repalced by "". Otherwise I think it end up as 'None' as default value.

Also, they would need to be documented in doc/library/config.txt